test/bigtest/interactors/uploading-jobs-display.js

```diff
@@ -1,5 +1,38 @@
-import { interactor } from '@bigtest/interactor';
+import {
+  interactor,
+  collection,
+  hasClass,
+  scoped,
+  text,
+} from '@bigtest/interactor';
 
-@interactor class UploadingJobsDisplay {}
+import CalloutInteractor from '@folio/stripes-components/lib/Callout/tests/interactor';
+import ConfirmationModalInteractor from '@folio/stripes-components/lib/ConfirmationModal/tests/interactor';
 
+import css from '../../../src/components/UploadingJobsDisplay/components/FileItem/FileItem.css';
+
+@interactor
+class LandingPageLink {
+  static defaultScope = '#ModuleMainHeading';
+}
+
+@interactor class FileItem {
+  hasUploadingClass = hasClass(css.fileItemUploading);
+  hasDangerClass = hasClass(css.fileItemDanger);
+  hasFailedClass = hasClass(css.fileItemFailed);
+  errorMessage = text('[data-test-error-message]');
```

Comment:
Let's name it `errorMsg` and rename attribute to be `data-test-error-msg` for consistency or rename existing ones for consistency.